src/components/Greeting.js

```diff
@@ -0,0 +1,9 @@
+function Greeting({title,link}){
+  return(
+    <div className="flex flex-col justify-center items-center">
+      <h1 className="text-3xl my-4 md:text-6xl lg:text-12xl">{title}</h1>
+      <a href="#" className="text-main-green">{link}</a>
```

Comment:
Where the link points to should also be a param. (even if whats being passes in right now is just "#")